pages/guides/sandbox/customers.mdx

```diff
@@ -0,0 +1,84 @@
+---
+layout: guides
+title: "Customers"
+guide:
+  step: 1
+meta:
+  title: "Testing Different API Customer Types | Dwolla API Documentation"
+  description: Test and simulate adding Customers and handling verification statuses in the Sandbox before working with real end users.
+---
+
+import imageFail from "../../../assets/images/content-images/test-document-upload-fail.png";
+import imageSuccess from "../../../assets/images/content-images/test-document-upload-success.png";
+import conceptsIcon from "../../../assets/images/content-images/content-icons/concepts-icon.svg";
+import guidesIcon from "../../../assets/images/content-images/content-icons/guides-icon.svg";
+
+# Testing Customers
+
+#### Manage Customers
```

Comment:
Should we change this to an h3 and title it `Manage Customers in the Dashboard` ?

Reply:
Resolved! - [767d461](https://github.com/Dwolla/dev-portal/pull/433/commits/767d461dc4bb4844e0e90e2ec78ef3cb2fd8dd8b)